product-service/tests/handler.test.ts

```diff
@@ -0,0 +1,25 @@
+import {APIGatewayEvent, Context} from "aws-lambda";
+import * as handler from "../handler";
+
+test("getProductsList", async () => {
```

Comment:
It will be nice to add some tests for catch part of handler (but not necessary for this task)